pages/_app.tsx

```diff
@@ -0,0 +1,34 @@
+import App from "next/app";
+import * as React from "react";
+import { ThemeProvider } from "emotion-theming";
+import { Global } from "@emotion/core";
+import { ThemeDecorator, BaseTheme } from "@mojotech/mojo-ui";
+
+export default class extends App {
+  render() {
+    const { props } = this as any;
+    const { Component, pageProps } = props;
```

Comment:
These 2 lines can be cleaned up and we can get rid of the `any` while we're at it

```suggestion
    const { Component, pageProps }: AppProps = this.props;
```